topics/assembly/tutorials/ERGA-post-assembly-QC/workflows/Post-assembly-workflow-tests.yml

```diff
@@ -0,0 +1,120 @@
+- doc: Test outline for Post-assembly-workflow
+  job:
+    Metadata file:
+      class: File
+      path: test-data/Metadata file.yaml
+      filetype: yaml
+    NCBI taxdump directory:
+      class: File
+      path: https://zenodo.org/record/7781236/files/new_taxdump.tar.gz
+      filetype: gz
+    Primary genome assembly file (fasta):
+      class: File
+      path: https://zenodo.org/record/7831298/files/assembly_sponge.fasta.gz
+      filetype: fasta.gz
+    DIAMOND database:
+      class: File
+      path: https://zenodo.org/record/7788734/files/Diamond_makedb.dmnd
+      filetype: dmnd
+    Long-read FASTQ files:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: dataset_01
+        path: https://zenodo.org/record/7786773/files/hifi.fastq.gz
+    Hi-C reverse:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: '0'
+        path: https://zenodo.org/record/7786773/files/hiC_2.fastq.gz
+    Hi-C forward:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: '0'
+        path: https://zenodo.org/record/7786773/files/hiC_1.fastq.gz
+    NCBI taxonomic ID: '121349'
+    Ploidy for model to use: '2'
+  outputs:
+    log_plot (Step 36):
+      path: test-data/log_plot (Step 36).png
```

Comment:
PNGS and JPGs can be moved to Zenodo. Else, everytime you need to compress them in order to reduce file size